cartography/intel/aws/lambda_function.py

```diff
@@ -105,6 +119,7 @@ def load_lambda_functions(
     lambda.architectures = lf.Architectures,
     lambda.masterarn = lf.MasterArn,
     lambda.kmskeyarn = lf.KMSKeyArn,
+    lambda.functionurl = lf.FunctionUrl,
```

Comment:
I don't see where this property is populated in `lf`. Can you highlight the code where it's populated?